File: 7_kyu/Python/isogram.py
def is_isogram(string):
    string = string.lower()
    i = 0
    while i < len(string):
        j = 0
        while j < len(string):
            if i != j and string[i] == string[j]:
                return False
            j += 1
        i += 1
    return True


print(is_isogram("Dermatoglyphics"))   #, True )
print(is_isogram("isogram"))   #, True )
print(is_isogram("aba"))   #, False, "same chars may not be adjacent" )
print(is_isogram("moOse"))  # False, "same chars may not be same case" )
print(is_isogram("isIsogram"))   #, False )
print(is_isogram(""))   #, True, 

File: 7_kyu/JavaScript/isogram.js
function isIsogram(string) {
    lstr = string.toLowerCase();
    for (let i = 0; i < lstr.length; i++) {
        for (let j = 0; j < lstr.length; j++){
            if (i != j && lstr[i] == lstr[j])
                return false;
        }
    }
    return true;
}

console.log(isIsogram("Dermatoglyphics"))   //, True )
console.log(isIsogram("isogram"))   //, True )
console.log(isIsogram("aba"))   //, False, "same chars may not be adjacent" )
console.log(isIsogram("moOse"))  // False, "same chars may not be same case" )
console.log(isIsogram("isIsogram"))   //, False )
console.log(isIsogram(""))   //, True, 